### PR TITLE
[18.01] Release Notes

### PR DESCRIPTION
**Pull request by martenson**
There will be at least 460 PRs in 18.01. By this metric this is the biggest Galaxy release to this date.

ping @galaxyproject/core - feel free to modify as you see fit

### CODE REVIEW

**Comment by jmchilton** [comment]
Note to self - we need a link for 18.01 docs but Jenkins isn't generating them yet.

**Comment by martenson** [comment]
@jmchilton we should probably add docs building to the process at the start of the release

File: doc/source/releases/18.01.rst [hunk 1]
```diff
@@ -0,0 +1,1359 @@
...
+.. feature
+
+
+* Clone tools in workflows
+  (thanks to `@anatskiy <https://github.com/anatskiy>`__).
+  `Pull Request 5087`_
+* Cloud ObjectStore
+  (thanks to `@VJalili <https://github.com/VJalili>`__).
+  `Pull Request 4487`_
+* Reload tools immediatelly if a macro changes
```
Comment (nsoranzo) [inline doc/source/releases/18.01.rst]: s/immediatelly/immediately/

File: doc/source/releases/18.01_announce.rst [hunk 2]
```diff
@@ -3,8 +3,125 @@
...
 
-Schedule
+Highlights
 ===========================================================
- * Planned Freeze Date: 2018-01-08
- * Planned Release Date: 2018-01-29
+
+**Performance and User Experience Improvements**
+  We made Galaxy more lively and responsive. Homepage, published workflows, published/saved histories, and data libraries should all load much faster now. Importing data from FTP will also take less of your time.
+  We achieved this by optimising queries, implementing cache, rethinking presentation, and adding progress bars and loading indicators where needed.
```
Comment (nsoranzo) [inline doc/source/releases/18.01_announce.rst]: optimising database queries

File: doc/source/releases/18.01_announce.rst [hunk 3]
```diff
@@ -3,8 +3,125 @@
...
+
+**Web Server and Configuration**
+  The default web server used by Galaxy has changed from Paste to uWSGI and the default configuration file for Galaxy is now ``config/galaxy.yml`` instead of ``config/galaxy.ini``. 
+  uWSGI is more production ready and should allow Galaxy to scale better in its default
+  configuration. In the future uWSGI will allow Galaxy to setup GIE proxies without additional
+  configuration and use modern web technology such as web sockets.
```
Comment (nsoranzo) [inline doc/source/releases/18.01_announce.rst]: s/technology/technologies/

File: doc/source/releases/18.01_announce.rst [hunk 4]
```diff
@@ -3,8 +3,125 @@
...
+  by setting ``APP_WEBSERVER=uwsgi`` in the environment). As part of the transition to YAML-based 
+  configuration files we have implemented a schema to validate Galaxy configuration files. Run
+  ``make config-validate`` from Galaxy's root directory to validate a schema and ``make config-lint`` to
+  check for best practices. While there is no need to convert your configuration file (``galaxy.ini`` hasn't
+  been deprecated), we have included make targets ``make config-convert-dry-run`` and ``make config-convert``
+  to assist converting an ``ini`` configuration file to a YAML one.
+
+  These are big changes that affect many parts of Galaxy's administration documentation and makes
+  this documentation very dependent on which Galaxy version they are targetting. To address this, we have
+  moved a signifcant amount of administration documentation into Galaxy's code
```
Comment (nsoranzo) [inline doc/source/releases/18.01_announce.rst]: s/signifcant/significant/

File: doc/source/releases/18.01_announce.rst [hunk 5]
```diff
@@ -3,8 +3,149 @@
...
+  be found `here <https://docs.galaxyproject.org/en/release_17.09/admin/index.html>`__, while documentation for this release can be found `here <https://docs.galaxyproject.org/en/release_18.01/admin/index.html>`__.
+
+  `Pull Request 4475`_, `Pull Request 5135`_, `Pull Request 5390`_, `Pull Request 5373`_,
+  `Pull Request 5105`_, `Pull Request 5441`_
+
+**Client Architecture**
+  The architecture for the client code that powers the Galaxy user interface has been significantly
+  overhauled. The code base has been converted to `ES6 <http://es6-features.org/>`__,
+  `Yarn <https://github.com/yarnpkg/yarn>`__ now powers the build and dependency management of the code,
+  `Prettier <https://prettier.io/>`__ is now used ensure consistent code formatting,
```
Comment (nsoranzo) [inline doc/source/releases/18.01_announce.rst]: to ensure

File: doc/source/releases/18.01_announce.rst [hunk 6]
```diff
@@ -3,8 +3,149 @@
...
+**Experimental Job Caching**
+  Galaxy can now be configured to allow users the option of skipping duplicated jobs if one with
+  identical parameters has been previously executed and simply reuse the previously generated outputs.
+  This contribution is thanks to `@mvdbeek <https://github.com/mvdbeek>`__.
+
+  `Pull Request 4690`_
+
+**Dataset Collection Usability**
+  This release has significantly improved the usability of Galaxy dataset collections. Dozens of improvements
+  to collections have been made, some the key highlights include:
```
Comment (nsoranzo) [inline doc/source/releases/18.01_announce.rst]: some of

**Comment by martenson** [comment]
I think we are good to go, I set up the job for docs building so after this is merged we can get some more feedback from folks who dislike reading rst.

**Comment by martenson** [comment]
Danke für der merge @natefoo

**Pull request merged**